src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  const values = Object.values(robot);
+
+  for (let i = 0; i < values.length; i++) {
+    for (let j = i + 1; j < values.length; j++) {
+      if (values[i] === values[j]) {
+        return null;
+      } else {
+        for (const key in robot) {
+          inverse[robot[key]] = key;
+        }
+      }
+    }
+  }
```

Comment:
let's rewrite it just using `for of` (`Object.keys(robot)`) and checking if the property already exists in `inverse` with `.hasOwnProperty()`